operator/controllers/servicerun_controller.go

```diff
@@ -74,14 +74,7 @@ func (r *ServiceRunReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return reconcile.Result{}, fmt.Errorf("could not fetch ServiceRun: %+v", err)
 	}
 
-	logger.Info("Searching for service")
-
-	service := &v1alpha1.Service{}
-	err = r.Get(ctx, types.NamespacedName{Name: serviceRun.Spec.ServiceName, Namespace: serviceRun.Namespace}, service)
-	if err != nil {
-		return reconcile.Result{}, fmt.Errorf("could not fetch Service: %+v", err)
-	}
-
+	// check for post-deployment checks
```

Comment:
please adapt the comment to indicate that the check for the referenced replicaSet/pod will be added here later. Otherwise this comment might be confusing right now